### DISCUSSION TITLE
umi4引入其他库的样式怎么写呢

### DISCUSSION THREAD

**Discussion by huxx5429** [umijs/umi#12304] [Q&A]
const extraBabelPlugins = [
  [
    'babel-plugin-import',
    {
      libraryName: '@turing-design/react-components',
      libraryDirectory: 'es',
      style: true,
    },
  ],
];

umi3里面是这样写的，升级到umi4之后就有报错
TypeError: Cannot read properties of undefined (reading 'dispose')

**Comment by fz6m** [comment]
感觉不是这个插件带来的问题吧，你有最小报错的复现吗。
